docs/csharp/programming-guide/statements-expressions-operators/snippets/how-to-test-for-reference-equality-identity/Program.cs

```diff
@@ -71,7 +71,7 @@ static void Main()
             // After a new string is assigned to strA, strA and strB
             // are no longer interned and no longer have reference equality.
             strA = "Goodbye world!";
-            Console.WriteLine("strA = \"{0}\" strB = \"{1}\"", strA, strB);
+            Console.WriteLine($"strA = \"{strA}\" strB = \"{strB}\"");
```

Comment:
Consider raw string literals (Yes, the parser should handle the 4 repeated quotes correctly.)

```suggestion
            Console.WriteLine($"""strA = "{strA}" strB = "{strB}"""");
```